### ISSUE TITLE
globoplay.globo.com: breakage

### ISSUE THREAD

**Issue by allanjales** [uBlockOrigin/uAssets#22895]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection or [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://globoplay.globo.com/tv-globo/ao-vivo/`

### Category

breakage

### Description

Globoplay is a television site on Brazil. I notice that when I am watching without uBlock Origin, it allows me to rewind on live broadcast, but when activating uBlock Origin, it is not possible.

[Here is the link for online service](https://globoplay.globo.com/tv-globo/ao-vivo/)

### Other extensions used

Bionic Reading
Language Tools
FPS extension
I don't care about cookies
Imagus
Left Right Sound Select
Méliuz: Cashback e cupons de desconto
Picture-in-Picture Extension (by Google)
Search by Image
Tampermonkey (no script in this page)
TLDR This
Wayback Machine

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

With extension active is not possible to go back in time (my cursor is over the red line):
![With extension active is not possible to go back in time](https://github.com/uBlockOrigin/uAssets/assets/30681577/915c64be-47e2-4f22-b871-21944477c09a)

Disabling uBlock Origin in this website I can go back in time (my cursor is over the red line):
![Disabling uBlock Origin in this website I can go back in time](https://github.com/uBlockOrigin/uAssets/assets/30681577/c3b13aeb-9957-4332-8b52-dc03e0e3904b)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Chromium: 122
filterset (summary):
 network: 126414
 cosmetic: 49332
 scriptlet: 20695
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 18-0, never
  easylist: 82223-15, 43m Δ
  easyprivacy: 50358-8, 43m Δ
  plowe-0: 3772-1184, now
  spa-1: 5435-71, now
  ublock-badware: 7858-147, 43m Δ
  ublock-filters: 37498-370, 43m Δ
  ublock-privacy: 756-6, 43m Δ
  ublock-quick-fixes: 150-4, 43m Δ
  ublock-unbreak: 2244-38, 43m Δ
  urlhaus-1: 8125-0, now
filterset (user): [array of 18 redacted]
trustedset:
 added: [array of 3 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 319 ms (selfie)
 maxAssetCacheWait: 106 ms
 cacheBackend: browser.storage.local
popupPanel:
 blocked: 36
 network:
  globo.com: 17
  cedexis.com: 1
  doubleclick.net: 1
  googletagmanager.com: 3
  gstatic.com: 1
  imasdk.googleapis.com: 2
  scorecardresearch.com: 7
  youborafds01.com: 4
 extended:
  ##+js(no-setTimeout-if, f.parentNode.removeChild(f), 100)
```
</details>

**Comment by D4niloMR** [comment]
Test:

```adb
@@||imasdk.googleapis.com/js/sdkloader/ima3_dai.js$script,domain=globoplay.globo.com
```
Click on uBO icon > :gear: [Dashboard button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Add the filter(s) in ["My filters" pane](https://github.com/gorhill/uBlock/wiki/Dashboard:-My-filters) > ✓ Apply changes

**Comment by YoshiTabletopGamer** [comment]
Can't reproduce breakage

**Comment by allanjales** [comment]
Sorry for my late reply, I was not able to get my PC those days.

I have applied the suggested filter, and it seems to work well. Thank you!

For now the problem is fixed.
